sbt-plugin/src/main/scala/scala/scalajs/sbtplugin/ScalaJSPluginInternal.scala

```diff
@@ -575,7 +576,11 @@ object ScalaJSPluginInternal {
           (tf, new ScalaJSFramework(name, env, classpath, logger, console))
         }
       }
-  )
+    ) ++ inConfig(Test)(Seq(
```

Comment:
`inConfig(Test)(Seq(` is redundant. `scalaJSTestFrameworkSettings` itself passes through an `inConfig(Test)` filter down the line.